### PR TITLE
Add namecheap component

### PR DESCRIPTION
**Pull request by Munsio**
**Description:**

Add the component site for the namecheapDNS

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#9821

### CODE REVIEW

File: source/_components/namecheapdns.markdown [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+<p class='note warning'>
+Namecheap only supports IPv4 adresses to update.
+</p>
+
+To use the component in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+namecheapdns:
```
Comment (fabaff) [inline source/_components/namecheapdns.markdown]: Rename this to `subdomain` please. This looks like a hack at the moment.

Comment (balloob) [reply source/_components/namecheapdns.markdown]: The component is called namecheapdns?

Comment (Munsio) [reply source/_components/namecheapdns.markdown]: Is there something to change or is this comment resolved cause the PR is already merged!?

Comment (fabaff) [reply source/_components/namecheapdns.markdown]: Sorry, wrong line. i was referring to `host`.

Comment (Munsio) [reply source/_components/namecheapdns.markdown]: At the namecheap doc site they reference to the parameter as host so i thought if you use the component in addition to the documentation from namecheap you will know what you have to fill in there.

The only thing i turned around was the "access_token" they reference it as "dynamic DNS Password"

**Pull request merged**